irteus/irtgeo.l

```diff
@@ -466,9 +471,26 @@
            (mapcan #'(lambda (e)
                        (let ((p (geo::line-intersection3 (e . pvert) (e . nvert)
                                                          (car v) (cdr v))))
-                         (when (and (<= 0.0 (car p) 1.0)
-                                    (< 0.0 (cadr p) 1.0))
-                           (list e))))
+                         (cond ((and (<= 0.0 (car p) 1.0)
+                                     (< 0.0 (cadr p) 1.0))
+                                (list e))
+                               ((and (< 0.0 (car p) 1.0)
```

Comment:
add comments on each condition.

Reply:
added https://github.com/euslisp/jskeus/pull/585/commits/5fb6cee646f0e838e32031554978685b38f527e5

`(butlast rvers)` is not the same as `rest-polygon` so the intersection is not detected correctly.
![butlast](https://user-images.githubusercontent.com/32383525/122324533-e5db5280-cf63-11eb-9497-e67e2d2acaaa.png)

Reply:
added https://github.com/euslisp/jskeus/pull/585/commits/b7cdaf2a45a4f254cf9fc29c75df2d71d73caf6e